dissect/volume/md/stream.py

```diff
@@ -0,0 +1,429 @@
+from __future__ import annotations
```

Comment:
Seems like this file's functionality is (almost) completely superseded by `raid/stream.py`?

Reply:
Oops yeah, this file was supposed to have been deleted.